### PR TITLE
Add TensorFlow

### PR DESCRIPTION
**Pull request by jtoy**
a list of awesome Tensorflow items

### CODE REVIEW

**Comment by davisonio** [comment]
Thanks for the suggestion :smiley: 
Please have a quick read through of the [instructions](https://github.com/sindresorhus/awesome/blob/master/create-list.md) for creating a new list.

Could you add a license (CC0 preferably) and some contribution guidelines? You can use [this one](https://github.com/sindresorhus/awesome/blob/master/contributing.md) as a template. Also, this pull request: https://github.com/jtoy/awesome-tensorflow/pull/3 hasn't been reviewed yet.

Perhaps a small description and a link to tensorflow.org would be good for this project - a new user visiting wouldn't know what Tensorflow is. e.g. `Tensor flow is an open source software library for numerical computation using data flow graphs`.

**Comment by jtoy** [comment]
awesome suggestions , I've made the changes! let me know if you have other ideas. I update the list about once a week.

**Comment by aleksandar-todorovic** [comment]
I still don't see the license anywhere.

**Comment by jtoy** [comment]
Added

**Comment by sindresorhus** [comment]
Why should TensorFlow be under `Platform`?

The list could use some improving when it comes to formatting and grammar. You've pretty much have 5 different ways of writing TensorFlow in there. Please keep it consistent. Same with ending an item description in a `.` or not. It's `GitHub`, not `Github`.

With `TensorFlow Libraries`, the `TensorFlow` prefix is moot.

`If you want to contribute to this list (please do), send me a pull ` and following should not be under the `What is TensorFLow` section.

File: readme.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [iOS](https://github.com/vsouza/awesome-ios)
 - [Android](https://github.com/JStumpp/awesome-android)
 - [IoT & Hybrid Apps](https://github.com/weblancaster/awesome-IoT-hybrid)
+- [Tensorflow](https://github.com/jtoy/awesome-tensorflow/)
```
Comment (sindresorhus) [inline readme.md]: TensorFlow

and don't end the url in a `/`

Comment (sindresorhus) [inline readme.md]: Should also be added to the end of the section. Please actually read https://github.com/sindresorhus/awesome/blob/master/contributing.md ...

**Comment by jtoy** [comment]
@sindresorhus and all, thanks for the suggestions, I cleaned it all up.  I think ready to merge.

**Comment by jtoy** [comment]
@sindresorhus on the platform question, where do you suggest it goes under then?
Platform was the best category out of the current categories. TensorFlow is a platform for writing deep learning models.
Its not computer science or theory since its a software library.
maybe under programming language?
I personally would make a new category called machine learning and put  it under that.  What do you think?

**Comment by sindresorhus** [comment]
> I personally would make a new category called machine learning and put it under that. What do you think?

Could the following existing item go under that new category `Computer Vision`, ``Deep Learning (Neural networks)`, `Deep Vision`?

**Comment by jtoy** [comment]
I like under deep learning, will do!

**Comment by sindresorhus** [comment]
No need to open a new pull request. In the future just update the existing pull request. Opening duplicate pull requests just makes the issue tracker noisy. 

https://github.com/sindresorhus/awesome/blob/master/contributing.md#updating-your-pull-request

**Pull request closed without merging**